doc/dox_comments/header_files/aes.h

```diff
@@ -11,7 +11,8 @@
     \param len length of the key passed in
     \param iv pointer to the initialization vector used to initialize the key
     \param dir Cipher direction. Set AES_ENCRYPTION to encrypt,  or
-    AES_DECRYPTION to decrypt.
+    AES_DECRYPTION to decrypt. Direction for some modes (CFB, XTS, CTR) is
```

Comment:
Please remove XTS from the list, XTS does do streaming and is similar but the special XTS set key API takes decryption direction. When mentioning XTS internally I forgot that the XTS set key function handles setting the direction to encrypt internally.